configs/_base_/datasets/iSAID.py

```diff
@@ -0,0 +1,57 @@
+# dataset settings
+dataset_type = 'ISAIDDataset'
+data_root = 'data/iSAID'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+crop_size = (896, 896)
```

Comment:
896 or 512?

Reply:
It should be 896. Sorry for misleading number 512 in config names and docs.

Reply:
Have experiments with crop size 512.